tools/toolchain/image

```diff
@@ -1 +1 @@
-docker.io/scylladb/scylla-toolchain:fedora-41-20250209
+docker.io/scylladb/scylla-toolchain:fedora-41-20250223
```

Comment:
The patch says nothing about which optimized clang binaries were used and where one can obtain them.

Reply:
updated